chore(auth): update auth config doc for re-init (#320)
apache/incubator-hugegraph-doc: content/en/docs/config/config-authentication.md

If deployed based on Docker image or if HugeGraph has already been initialized and needs to be converted to authentication mode, 
relevant graph data needs to be deleted and HugeGraph needs to be restarted. If there is already business data in the diagram, 

> it is temporarily **not possible** to directly convert the authentication mode (improvements/support for this feature will be released in the next version. The latest code has already been fixed: [PR 2411](https://github.com/apache/incubator-hugegraph/pull/2411).).

Reviewer: ```suggestion
it is temporarily **not possible** to directly convert the authentication mode (improvements/support for this feature will be released in the next version. Could refer [PR 2411](https://github.com/apache/incubator-hugegraph/pull/2411)).
```
